component css/grid.css

```diff
@@ -0,0 +1,68 @@
+.grid-first{
+    grid-area: grid-first;
+    background-color: aqua;
+    height: 3rem;
+}
+.grid-second{
+    grid-area: grid-second;
+    height: 3rem;
+    background-color: aquamarine;
```

Comment:
it's better if you put non-CL classes into a separate file like `docs.css`

Reply:
its component library css not for my webpage.